pages/mesosphere/dcos/services/data-science-engine/1.0.1/release-notes/index.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes
+excerpt: Release notes for DC/OS Data Science Engine
```

Comment:
How about this: `Discover the new features, updates, and known limitations in this release of the {{ model.techName }}`

Reply:
Done. The header metadata cannot contain or resolve variables, so I just added the name of the product.